cmd/data-node/commands/dehistory/show.go

```diff
@@ -65,6 +67,19 @@ func (cmd *showCmd) Execute(_ []string) error {
 		fmt.Printf("%s\n", segment)
 	}
 
+	contiguousHistory := GetHighestContiguousHistoryFromHistorySegments(segments)
+	// Sort history oldest first
+	sort.Slice(contiguousHistory, func(i, j int) bool {
```

Comment:
Isn't it already sorted by now? `GetHighestContiguousHistoryFromHistorySegments` returns it sorted (via `GetHighestContiguousHistory`)

Reply:
no, the method returns latest first